src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -167,13 +167,13 @@ public void updateParticipant(Id id, Participant participant) throws AlfredExcep
             this.participantList.update(id, participant);
             return;
         }
+        this.participantList.update(id, participant);
+
         boolean isSuccessful = targetTeam.updateParticipant(participant);
         if (!isSuccessful) {
             logger.warning("The participant is not in the team provided");
```

Comment:
If you get the team by participant id, why would the participant not be in that team? And so we're allowing the user to update the participant even if he/she's not in a team right?

Reply:
For the first one, maybe the participant does not belong to a team or has not been assigned. And yes to the second, we always allow for the participant in the participant list to be updated

Reply:
Cool I was just wondering about `if (!isSuccessful)` part